pkg/apis/planetscale/v2/etcdlockserver_types.go

```diff
@@ -126,6 +126,42 @@ type EtcdLockserverTemplate struct {
 	// ExtraLabels can optionally be used to attach custom labels to Pods
 	// created for this component.
 	ExtraLabels map[string]string `json:"extraLabels,omitempty"`
+
+	// CreatePDB sets whether to create a PDB for etcd member Pods.
```

Comment:
spell out what PDB means
```suggestion
	// CreatePDB sets whether to create a PodDisruptionBudget (PDB) for etcd member Pods.
```

Reply:
Done.